versions/v6/header-front.php

```diff
@@ -21,47 +21,47 @@
 
 		<header class="front-page-header">
 			<div class="container">
-				<div class="row hidden-xs">
-					<div class="col-sm-12">
-						<div class="fp-social-links pull-right">
+				<div class="row d-none d-md-block">
+					<div class="col-md-12 d-flex">
+						<div class="fp-social-links ml-auto">
 							<?php echo display_social_header(); ?>
 						</div>
 					</div>
 				</div>
 
 				<div class="row">
-					<div class="col-xs-6 col-sm-3 col-md-2 text-center">
+					<div class="col-6 col-md-3 col-lg-2 text-md-center">
 						<?php 						$current_issue = get_current_issue();
 						$current_issue_title = wptexturize( $current_issue->post_title );
 						?>
-						<a class="fp-header-link fp-header-link-issue hidden-xs" href="<?php echo get_permalink( $current_issue->ID ); ?>">
+						<a class="fp-header-link fp-header-link-issue d-none d-md-block" href="<?php echo get_permalink( $current_issue->ID ); ?>">
 							<?php echo str_replace(' ', '<br>', $current_issue_title); ?>
 						</a>
-						<a class="fp-header-link fp-header-link-issue visible-xs-block" href="<?php echo get_permalink( $current_issue->ID ); ?>">
+						<a class="fp-header-link fp-header-link-issue d-md-none" href="<?php echo get_permalink( $current_issue->ID ); ?>">
 							<?php echo $current_issue_title; ?>
 						</a>
 					</div>
-					<div class="col-xs-6 col-sm-3 col-sm-push-6 col-md-2 col-md-push-8 text-center">
+					<div class="col-6 col-md-3 flex-md-last col-lg-2 text-right text-md-center">
 						<a class="fp-header-link fp-header-link-archives" href="<?php echo get_permalink( get_page_by_title( 'Archives' ) ); ?>">Archives</a>
 					</div>
-					<div class="clearfix visible-xs-block"></div>
-					<div class="col-sm-6 col-sm-pull-3 col-md-8 col-md-pull-2 text-center">
+					<div class="clearfix d-md-none"></div>
```

Comment:
I don't think this clearfix div is necessary, I deleted it out in dev (inspector tools) and everything seems to function correctly, so I think we can remove it.